src/WebJobs.Script/Description/Node/NodeFunctionInvoker.cs

```diff
@@ -270,8 +270,8 @@ protected override void OnScriptFileChanged(object sender, FileSystemEventArgs e
                 string text = p as string;
                 if (text != null)
                 {
-                    traceWriter.Verbose(text);
-                    fileTraceWriter.Verbose(text);
+                    traceWriter.Info(text);
```

Comment:
Whereas for C# the user can control the log level (since they're binding to a TraceWriter), for Node we write all log statements currently as Info.